src/mailadm/cmdline.py

```diff
@@ -50,6 +58,88 @@ def get_mailadm_db(ctx, show=False, fail_missing_config=True):
     return db
 
 
+@click.command()
+@click.option("--email", type=str, default=None, help="name of email")
+@click.option("--password", type=str, default=None, help="name of password")
+@click.option("--show-ffi", is_flag=True, help="show low level ffi events")
+@click.pass_context
+@account_hookimpl
+def setup_bot(ctx, email, password, show_ffi):
+    """initialize the deltachat bot as an alternative command interface.
+
+    :param ctx: the click object passing the CLI environment
+    :param email: the email account the deltachat bot will use for receiving commands
+    :param password: the password to the bot's email account
+    :param db: the path to the deltachat database of the bot - NOT the path to the mailadm database!
+    :param show_ffi: show low level ffi events
+    """
+    admbot_db = get_admbot_db_path()
+    ac = Account(admbot_db)
+    if show_ffi:
+        ac.add_account_plugin(FFIEventLogger(ac))
+
+    if not ac.is_configured():
+        assert email and password, (
+            "you must specify --email and --password once to configure this database/account"
+        )
+    if email:
+        ac.set_config("addr", email)
+    if password:
+        ac.set_config("mail_pw", password)
+    ac.set_config("mvbox_move", "0")
+    ac.set_config("sentbox_watch", "0")
+    ac.set_config("bot", "1")
+    configtracker = ac.configure(reconfigure=ac.is_configured())
+    configtracker.wait_finish()
+
+    ac.start_io()
+
+    mailadmdb = get_mailadm_db(ctx)
+    with mailadmdb.read_connection() as rconn:
+        mail_domain = rconn.config.mail_domain
+        admingrpid_old = rconn.config.admingrpid
+
+    chat = ac.create_group_chat("Admin group on {}".format(mail_domain), contacts=[], verified=True)
```

Comment:
```suggestion
    chat = ac.create_group_chat("AAdmins {}".format(mail_domain), contacts=[], verified=True)
```
"Account Admins" :)

Reply:
hmmm, not a fan^^ I mean, users can rename it as they want, but I called it an admin group to make obvious what it's for...